easybuild/easyconfigs/m/MathGL/MathGL-2.4.1_FL.patch

```diff
@@ -0,0 +1,66 @@
+# include direcotry is all caps
```

Comment:
clarify that this is the `FLTK` include directory (and fix the typo in `direcotry` while you're at it) ;)

don't forget to update the `SHA256` checksum for the patch if you change it